2-Browsers/Week1/test-reports/ex5-catWalk.fail.txt

```diff
@@ -0,0 +1,3 @@
+*** Unit Test Error Report ***
+
+- catWalk should use `setInterval()`
```

Comment:
The unit test checks for `setInterval()` but failed to detect `window.setInterval()`. I will fix this so that both forms are recognised for the benefit of the next class.